spinn_front_end_common/utilities/utility_calls.py

```diff
@@ -57,7 +57,7 @@ def get_region_base_address_offset(app_data_base_address, region):
     :param int region: the region ID we're looking for
     """
     return (app_data_base_address +
-            APP_PTR_TABLE_HEADER_BYTE_SIZE + (region * 4))
+            APP_PTR_TABLE_HEADER_BYTE_SIZE + (region * 4 * 3))
```

Comment:
Ideally the 4 and the 3 here should have named constants instead.

Reply:
Fixed